### PR TITLE
C23_WMDE_Desktop_DE_22

### PR DESCRIPTION
**Pull request by Abban**
- Based on the VAR of test 20
- VAR has a double progress bar

Ticket: https://phabricator.wikimedia.org/T353329

### CODE REVIEW

File: banners/desktop/banner_ctrl.ts [hunk 1]
```diff
@@ -59,7 +59,8 @@ app.use( DynamicTextPlugin, {
 	date: new Date(),
 	formatters: localeFactory.getFormatters(),
 	impressionCount,
-	translator
+	translator,
+	urgencyMessageDaysLeft: 45
```
Comment (gbirke) [inline banners/desktop/banner_ctrl.ts]: Is this a leftover from some experiment?

**Review by gbirke — APPROVED** [review]
Code looks good.
I'm wondering how you tested locally, because the local overrides in `src/environment/dev/CampaignParameterOverride.ts` prevented me from previeweing the proper values. Maybe you can comment out or delete most of the overrides there, so the local display uses real campaign values

File: banners/desktop/banner_ctrl.ts (hunk 1 above)
Comment (Abban) [reply banners/desktop/banner_ctrl.ts]: No, the `campaignDaySentence` shows a more urgent message if we're within `N` days of the end date. It defaults to 10 but they wanted to activate it early and 45 was the random number I picked that was greater than the current days left when it was activated.

**Comment by Abban** [comment]
> I'm wondering how you tested locally,

I didn't, I uploaded it and did the final test on Central Notice

**Pull request merged**